modules/apps/site-initializer/site-initializer-masterclass/site-initializer-masterclass-test/src/testFunctional/macros/Masterclass.macro

```diff
@@ -49,6 +49,10 @@ definition {
 		Click(locator1 = "Button#SUBMIT");
 	}
 
+	macro goBack {
+		RunScript(locator1 = "window.history.go(-1)");
```

Comment:
do we have a task to track do it well once they implement https://issues.liferay.com/browse/POSHI-302

Reply:
I've linked POSHI task with this Testing Story. But I'll create a task to not forget to update the test after poshi task is done.